### DISCUSSION TITLE
BiRealNet test issue

### DISCUSSION THREAD

**Discussion by FrancescoLoro** [larq/zoo#295] [Q&A]
Hi everyone and thanks for making public the fantastic larq library.
I am trying to reproduce the networks present in the larq zoo in order to customize them and make some comparisons.
I started from the BiRealNet and I was able to reproduce it exactly based both on your code on github and on the references of the summary present in: https://docs.larq.dev/zoo/api/literature/.
I have downloaded the pretrained weights using the functions on your code and correctly load them into the model.
To test the network I used a reduced version of ImageNet (about 50000 samples) applying preprocess I found here: https://github.com/larq/zoo/blob/2e76138b8be0d235a5d43059915366c39508f74b/tests/models_test.py#L16 to all images.
However the network always predicts the same class regardless of the sample presented to it, and I can't figure out why.
Could you please tell me if I'm doing it wrong somewhere? 
Thanks a lot for you help!

**Comment by AdamHillier** [comment]
Hi @FrancescoLoro, sorry for the delayed response!

Could you perhaps share the exact evaluation script that you're using to test birealnet? It will make it easier to work out what might be going wrong.

**Reply by FrancescoLoro** [reply]
Sorry for the delay, I managed to solve it 👍

**Reply by AdamHillier** [reply]
Ah okay, great to hear :)
